src/styles/blocks/_about-us.scss

```diff
@@ -0,0 +1,88 @@
+.about-us,
+.technology {
+
+  &__content {
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      gap: 20px;
+    }
+  }
+
+  &__title {
+    @extend %title;
+
+    margin-bottom: 30px;
+
+    @include onTablet {
+      font-size: 36px;
+      line-height: 46px;
+    }
+
+    @include onDesktop {
+      font-size: 56px;
+      line-height: 70px;
+    }
+  }
+
+  &__description {
+    margin: 20px 0 30px;
+
+    @include onTablet {
+      margin-bottom: 60px;
+      font-size: 16px;
+      line-height: 24px;
+    }
+  }
+
+  &__photo {
+    @extend %bg-center;
+
+    width: 100%;
+    height: 210px;
+    transition: 0.4s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.